BigCommerce/Integration/01_ManageBigCommerceConnection.md

```diff
@@ -0,0 +1,195 @@
+
+
+## Create BigCommerce connection automatically 
+
+Create the connection using the *BigCommerce* driver. You will not be able to configure any other settings until the connection is established.
+
+
+### Prerequisites
+
+- You have a *BigCommerce* customer account to log in to *BigCommerce*.
+- You have created a *BigCommerce* store. 
+- In the *Actindo Core1 Platform*, you have permission to create connections in *Omni-Channel*.
+
+
+### Procedure
+
+1. Log in to your *Actindo Core1* instance, on which you run the *Omni-Channel* module. To verify the successful creation later, it is important to be logged in.
+
+2. In a new browser window, open the *BigCommerce App Store* (https://www.bigcommerce.com/apps/) and log in to your *BigCommerce* account using the [Log in] button in the upper right corner.   
+  The *BigCommerce Actindo-sandbox* is opened.
+
+   ![BigCommerce](../../Assets/Screenshots/Channels/Settings/Connections/BigCommerce/BigCommerceEntryScreen.png "[BigCommerce]")
+
+2. Click the *Apps* folder.
+
+3. Click the *Marketplace* folder and select the *Actindo Core1 DOP Connector* app.   
+  The *Actindo Core1 DOP Connector* app entry is displayed.
+
+   ![Select app](../../Assets/Screenshots/Channels/Settings/Connections/BigCommerce/SelectApp.png "[Select app]")
+
+ 4. Click the *Actindo* logo.   
+  The app is selected. Further information is provided. 
+
+    ![Actindo Onboarding app](../../Assets/Screenshots/Channels/Settings/Connections/BigCommerce/DownloadOnboardingApp.png "[Actindo Onboarding app]")
+
+5. Click the [Install] button.  
+  The site for confirming the access is opened.
+
+   ![Confirm access](../../Assets/Screenshots/Channels/Settings/Connections/BigCommerce/ConfirmAccess.png "[Confirm access]")
+
+  
+6. Click the [Confirm] button.   
+  The *BigCommerce installation* window is opened.
+
+   ![BigCommerce installation](../../Assets/Screenshots/Channels/Settings/Connections/BigCommerce/EnterWorkspaceURL.png "[BigCommerce installation]")
+
+7. Enter the workspace URL of the instance, on which your *Omni-Channel* module is running and click the [START SETUP] button.
+   If the installation was successful, the *BigCommerce connected* window is displayed.
+   In your instance, a new *Omni-Channel* connection is created, which is connected to your *BigCommerce* shop.   
+   
+   ![Connected to BigCommerce](../../Assets/Screenshots/Channels/Settings/Connections/BigCommerce/ConnectedToBigCommerce.png "[Connected to BigCommerce]")
+
+8. Change to your *Actindo Core1* instance and select *Omni-Channel > Settings > Tab CONNECTIONS*.   
+    - The new connection has been created, synchronized, and activated.   
+    - The credentials of the connection have been automatically created.
```

Comment:
credentials are automatically created oder eher filled in...?

Reply:
nö